### ISSUE TITLE
Map no activate on initial load

### ISSUE THREAD

**Issue by YSBDesign** [stowball/jQuery-rwdImageMaps#45]
Hi! i'm seeing your work its pretty cool!

I have a problem: when i open the page at first time there is a problem with the map; it isnt working only if i reescale the page it start working.

I saw the issue #2 but it isnt working for me...

Please help me!!!
Thanks!
YSB

**Comment by kiv5** [comment]
That's right. I'm having the same problem. I didn't see this thread and created a new issue for this. Any solutions?
